mipengine/node/monetdb_interface/views.py

```diff
@@ -15,6 +18,11 @@ def get_view_names(context_id: str) -> List[str]:
     return get_table_names(TableType.VIEW, context_id)
 
 
+@sql_injection_guard("view_name", isidentifier)
+@sql_injection_guard("table_name", is_primary_data_table)
+@sql_injection_guard("columns", isidentifier)
+# TODO https://team-1617704806227.atlassian.net/browse/MIP-673
+# @sql_injection_guard("filters", isidentifier)
```

Comment:
In SQL, if names are quoted then there is no risk of SQL injection, as long as we make sure that SQL producing functions always quote these fields and that these parameters (filters, literals, etc.) do not contain a quote or double quote. For example, this is not an SQL injection:
```sql
SELECT "DROP TABLE lalala" FROM TABLE lalala;
```
It just looks for a column with the name "DROP TABLE lalala".

The same applies for names appearing in the WHERE clause etc.